substrate/client/offchain/src/api/http.rs

```diff
@@ -48,21 +48,26 @@ use std::{
 
 const LOG_TARGET: &str = "offchain-worker::http";
 
+pub type Body = BoxBody<hyper::body::Bytes, hyper::Error>;
+
+type Sender = mpsc::Sender<Result<hyper::body::Frame<hyper::body::Bytes>, hyper::Error>>;
+type Receiver = mpsc::Receiver<Result<hyper::body::Frame<hyper::body::Bytes>, hyper::Error>>;
+
+type HyperClient = client::Client<HttpsConnector<client::connect::HttpConnector>, Body>;
+type LazyClient = Lazy<HyperClient, Box<dyn FnOnce() -> HyperClient + Send>>;
```

Comment:
Once rustc has stabilized TAIT, this boxing can be avoided.
The other option is implementing similar thing as `once_cell::Lazy` with `once_cell::OnceCell` and `std::cell::Cell`